students/markusiwerbo/lab2_part3/lab2_part3.ino

```diff
@@ -0,0 +1,36 @@
+//Lab 2 part 3
+//Distance sensor as analog input 
+
+const int input_pin = A0;   //Analog input from sensor
+const int led = 13;         //Digital utput pin
+const float voltage = 5.0;        //Sensor power supply
+const float voltage_per_in = voltage / 512.0; 
+ 
+const float half_inch_ref = 0.5; 
```

Comment:
Is the sensor reliable to this distance?

Reply:
The LED turned on at a larger distance. It stayed on at that distance.